spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java

```diff
@@ -219,9 +205,16 @@ public DeviceResult run(AndroidDebugBridge adb) {
       logDebug(debug, "About to grab screenshots and prepare output for [%s]", serial);
 
       // Sync device screenshots, if any, to the local filesystem.
-      String dirName = "app_" + SPOON_SCREENSHOTS;
+      String dirName = SPOON_SCREENSHOTS;
       String localDirName = work.getAbsolutePath();
-      final String devicePath = "/data/data/" + appPackage + "/" + dirName;
+
+      //Get external storage directory
```

Comment:
ok, suuuuper nit: please use a space after `//` and put a period at the end of the line. [Comments should be sentences](nedbatchelder.com/blog/201401/comments_should_be_sentences.html).